annotation/README.md

```diff
@@ -66,6 +66,24 @@ abstract class RestClient {
       {@QueryMap() Map<String, dynamic> queryies,
       @Field() int field,
       @Field("field-g") String ffff});
+
+  /// Do not forget to add the appropriate headers
+  @Headers({'Content-Type': 'multipart/form-data'})
```

Comment:
How about using a `Multipart` annotation instead?

Reply:
We can add another annotation sure. I will update the PR

Reply:
Actually dio automatically adds `Content-Type` header when we send form data. Look at this -> https://github.com/flutterchina/dio/blob/master/package_src/lib/src/dio.dart#L848-L851

So I will remove the `@Headers` from the example

Reply:
make sense.